spanner/transaction.go

```diff
@@ -1859,6 +1859,9 @@ type writeOnlyTransaction struct {
 	// current transaction from the allowed tracking change streams with DDL option
 	// allow_txn_exclusion=true.
 	excludeTxnFromChangeStreams bool
+	// maxCommitDelay is the maximum time that the commit will be delayed by the
+	// backend before it is acknowledged.
```

Comment:
(The field here is not `maxCommitDelay`, but `commitOptions`)

```suggestion
	// commitOptions are applied to the Commit request for the writeOnlyTransaction.
```